### DISCUSSION TITLE
Csv or data does not work with Wireless Connection

### DISCUSSION THREAD

**Discussion by Panuno24** [netbox-community/netbox#18144] [General]
Hi,

I see that when manually adding the place option it does appear but when adding importing, it is not there.

Is it a bug in this version?

![git](https://github.com/user-attachments/assets/763718a7-26ba-4d16-af7c-db0bf543f4d3)
![git2](https://github.com/user-attachments/assets/84145c5f-4aba-4ab3-86dc-90d5fb1fa784)
º
<class 'wireless.models.WirelessLink.interface_a.RelatedObjectDoesNotExist'>

WirelessLink has no interface_a.

Versión de Python: 3.11.2
Versión NetBox: 4.1.7
Plugins: No hay ninguno instalado

**Comment by candlerb** [comment]
It does appear to be broken.

When creating a Cable you'd create "side_a_device", "side_a_type", "side_a_name". I'd expect at very least device and interface to be specified for wireless links.

You're probably the first person to have tried doing this via CSV upload.

**Comment by Panuno24** [comment]
That was just what I thought, thanks.

**Comment by mblumber1** [comment]
Is there a way to do this import using yaml/json? I tried making my import look like the json that comes from the API, and I'm getting errors. If someone could provide an example yaml or json that has the a/b side device and interface in it that works with the tool, that would be greatly helpful
